### PR TITLE
docs for config profiles

### PR DESCRIPTION
**Pull request by jdgemm**
staging docs for config profiles

### CODE REVIEW

File: docs/use-the-network/console/profiles.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+id: profiles 
+sidebar_label: Configuration Profiles
+slug: /use-the-network/console/profiles
+---
+
+# Configuration Profiles
+
+[CURRENTLY ON STAGING ONLY]
```
Comment (kent-williams) [inline docs/use-the-network/console/profiles.mdx]: Did you mean to display brackets here?

Comment (jdgemm) [reply docs/use-the-network/console/profiles.mdx]: yes

File: docs/use-the-network/console/alerts.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+id: alerts
+sidebar_label: Alerts
+slug: /use-the-network/console/alerts
+---
+
+# Alerts
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Alerts (email or webhooks) can be configured for a number of conditions related to devices and integrations including when devices are deleted, integrations updated or deleted.
```
Comment (PharkMillups) [inline docs/use-the-network/console/alerts.mdx]:
```suggestion
Alerts (email or webhooks) can be configured for a number of conditions related to devices and integrations including when devices are deleted, or when integrations are updated or deleted.
```

File: docs/use-the-network/console/alerts.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+User types (e.g., Administrator or Managers) can be specified to receive alerts.
+
+For a video walkthrough of how to set up Alerts in Console 2.0, check out our Tips & Tricks video [here](https://youtu.be/8zt6W_FCQcM)
```
Comment (PharkMillups) [inline docs/use-the-network/console/alerts.mdx]:
```suggestion
For a video walkthrough of how to set up Alerts in Console 2.0, check out our Tips & Tricks video [here](https://youtu.be/8zt6W_FCQcM).
```

File: docs/use-the-network/console/alerts.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+### Configuring Alerts
+
+To configure alerts from the Flows workspace, select a Device/Label node or Integration node depending on the type of alert to configure and select the Inspector tab.
```
Comment (PharkMillups) [inline docs/use-the-network/console/alerts.mdx:19]:
```suggestion
To configure alerts from the Flows workspace, select a Device/Label node or Integration node depending on the type of alert you want to configure and select the Inspector tab. 
```

File: docs/use-the-network/console/alerts.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+To create a new one, select Create New Alert choose a name, alert type (email, webhook), and the alert. 
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/console/flows-inspector-alerts.png"
+  )}
+/>
+
+**Note:** for webhook alerts obtain a webhook key for authorization, by choosing Organizations from the side menu and copying relevant webhook key.
```
Comment (PharkMillups) [inline docs/use-the-network/console/alerts.mdx]:
```suggestion
**Note:** For webhook alerts, obtain a webhook key for authorization by choosing Organizations from the side menu and copying relevant webhook key.
```

File: docs/use-the-network/console/multi-packets.mdx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+id: multi-packets
+sidebar_label: Multiple Packets
+slug: /use-the-network/console/multi-packets
+---
+
+# Multiple Packets
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Data Credits are only charged for uplinks. The Multiple Packets feature provides the ability to pay for additional packets for redundancy and location triangulation purposes.
```
Comment (PharkMillups) [inline docs/use-the-network/console/multi-packets.mdx]:
```suggestion
By default, Data Credits are only charged for uplinks. The Multiple Packets feature provides the ability to pay for additional packets for redundancy and location triangulation purposes. 
```

File: docs/use-the-network/console/multi-packets.mdx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+# Multiple Packets
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Data Credits are only charged for uplinks. The Multiple Packets feature provides the ability to pay for additional packets for redundancy and location triangulation purposes. 
+
+To learn more about enabling Multiple Packets, check out our Tips and Tricks video [here](https://youtu.be/rDJcGspnq1A).
+
+**Note:** Regardless of the Multiple Packet feature setting, under certain conditions the system purchases multiple packets to help ensure successful packet transmission (these cost additional Data Credits and happen automatically). These include: initial joining of the device (up to 5 packets), downlinks that contain data payloads (up to 3 packets). These additional packets help ensure connection reliability to downlink via the Hotspot with the strongest connection.
```
Comment (PharkMillups) [inline docs/use-the-network/console/multi-packets.mdx]:
```suggestion
**Note:** Regardless of the Multiple Packet feature setting, under certain conditions the system purchases multiple packets to help ensure successful packet transmission (these cost additional Data Credits and happen automatically). These include: initial joining of the device (up to five packets) and downlinks that contain data payloads (up to three packets). These additional packets help ensure connection reliability to downlink via the Hotspot with the strongest connection.
```

File: docs/use-the-network/console/profiles.mdx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Configuration Profiles provide users with a streamlined way to configure, save, and apply settings to a device or group of devices (via labels).
+
+Unlike other configuration settings, Profiles impact LoRAWAN’s protocol behavior and are binary in nature. 
+
+:::important
+Configuration Profiles can be added to device/label nodes. The last Profile assigned to a device takes priority.  For example, if a device has Profile A, and a label has Profile B, if that label is applied to the device then Profile B overrides Profile A.
```
Comment (PharkMillups) [inline docs/use-the-network/console/profiles.mdx]:
```suggestion
Configuration Profiles can be added to Device/Label nodes. The last Profile assigned to a device takes priority.  For example, if a device has Profile A, and a label has Profile B, if that label is applied to the device then Profile B overrides Profile A. 
```

File: docs/use-the-network/console/profiles.mdx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+- [ADR](#adr)
+- [CF List](#cf-list) 
+
+:::info
+For a video walkthrough of configuring ADR and CF List, check out our Tips and Tricks video [here](https://youtu.be/9nlkAEI1V-Q).
+:::
+
+## ADR
+
+Adaptive Data Rate or ADR is a mechanism for adapting the data rate of devices
```
Comment (PharkMillups) [inline docs/use-the-network/console/profiles.mdx]:
```suggestion
Adaptive Data Rate (ADR) is a mechanism for adapting the data rate of devices
```

**Review by PharkMillups — CHANGES_REQUESTED** [review]
Made some small suggestions.

**Review by PharkMillups — APPROVED** [review]
Gucci

**Pull request merged**